app/src/main/java/com/example/sunmusic/screen/trending/TrendingPresenter.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.sunmusic.screen.trending
+
+import com.example.sunmusic.data.repository.AlbumRepository
+import com.example.sunmusic.data.repository.TrackRepository
+import com.example.sunmusic.utils.subscribe
+
+class TrendingPresenter(
+    private val albumRepository: AlbumRepository,
+    private val trackRepository: TrackRepository
+) : TrendingContract.Presenter {
+    private var view: TrendingContract.View? = null
+
+    override fun getTopAlbums(limit: Int) {
+        albumRepository.getTopAlbums(limit)
+            .subscribe({
+                view?.showTopAlbums(it)
+            }, {
+                view?.showError(it)
+            })
+    }
+
+    override fun getTopTracks(limit: Int) {
+        trackRepository.getTopTracks(limit)
+            .subscribe({
+                view?.showTopTracks(it)
+            }, {
+                view?.showError(it)
+            })
+    }
+
+    override fun onStart() = Unit
```

Comment:
hàm start này viết ra để làm gì trong khi ở trên View phải gọi liên tiếp hai hàm getTopAlbums và onTopTracks trong khi onStart để không 
nếu là mặc định limit thì có thể truyền thêm giá trị mặc định cho tham số của hàm 
```suggestion
    getTopAlbums(limit: Int = default_value)...
    getTopTracks(limit: Int = default_value)...
    override fun onStart() {
        getTopAlbums()
        getTopTracks()
    }
```